ush/forecast_postdet.sh

```diff
@@ -198,21 +198,46 @@ FV3_GFS_postdet(){
 		$NLN ${src_grid_spec} $DATA/INPUT/${CASE}_mosaic.nc
 	fi
 
-	if [ -z ${OROFIX} ]; then
-		if [ $FRAC_GRID ]; then
-			FIXoro=${FIXoro:-$FIX_DIR/fix_fv3_fracoro}/${CASE}.mx${OCNRES}_frac/${CASE}.mx${OCNRES}_
-		else
-			FIXoro=$FIXfv3/$CASE/${CASE}_oro_data.
-		fi
+	if [ $FRAC_GRID = ".true." ]; then
+		OROFIX=${OROFIX:-"${FIX_DIR}/fix_fv3_fracoro/${CASE}.mx${OCNRES}_frac"}
+		FIX_SFC=${FIX_SFC:-"${OROFIX}/fix_sfc"}
+		for n in $(seq 1 $ntiles); do
+			$NLN ${OROFIX}/oro_${CASE}.mx${OCNRES}.tile${n}.nc $DATA/INPUT/oro_data.tile${n}.nc
+		done
 	else
-		FIXoro=${OROFIX}/oro_data.
+		OROFIX=${OROFIX:-"${FIXfv3}/${CASE}"}
+		FIX_SFC=${FIX_SFC:-"${FIXgrd}/fix_sfc"}
+		for n in $(seq 1 $ntiles); do
+			$NLN ${OROFIX}/${CASE}_oro_data.tile${n}.nc $DATA/INPUT/oro_data.tile${n}.nc
+		done
 	fi
-	for n in $(seq 1 $ntiles); do
-		$NLN ${FIXoro}tile${n}.nc $DATA/INPUT/oro_data.tile${n}.nc
-	done
 
-	# GFS standard input data
+	_suite_file=$HOMEgfs/sorc/ufs_coupled.fd/FV3/ccpp/suites/suite_${CCPP_SUITE}.xml
```

Comment:
Should this statement be guarded? What happens if suite_${CCPP_SUITE}.xml does not exist?

Reply:
Other portions of the same script already use it, so I assumed it was guaranteed to be set. That does not appear to be the case after all. I will move the default from parsing_namelists_FV3.sh up somewhere to make sure it is set.

Reply:
And yes, should also check that the file exists.

Reply:
Fixed now